ci/terraform/oidc/wellknown.tf

```diff
@@ -4,6 +4,9 @@ module "openid_configuration_role" {
   role_name   = "openid-configuration"
   environment = var.environment
   vpc_arn     = local.authentication_vpc_arn
+  extra_tags = {
+    Service = "openid-configuration"
```

Comment:
Not clear to us how someone is going to use Cost Explorer and know that `openid-configuration` is the OAuth well known endpoint.

Reply:
as above